src/exercises/exerciseTypes/CountryFlags.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+function CountryFlags({ languageCode }) {
+    function getFlagEmoji(languageCode) {
+        const languageFlags = {
+            sq: "🇦🇱", // Albanian
+            ar: "🇪🇬", //Arabic, flag of Egypt, since it's the most commonly taught variant
+            "zh-CN": "🇨🇳", // Chinese""
+            da: "🇩🇰", // Danish
+            fr: "🇫🇷", // French
+            de: "🇩🇪", // German
+            el: "🇬🇷", // Greek
+            hu: "🇭🇺", // Hungarian
+            ind: "🇮🇩", // Indonesian 
+            it: "🇮🇹", // Italian
+            ku: <img src={"/static/icons/kurdish-flag.png"} alt="Kurdish flag" style={{height: '1em', width: '1em'}} />, //Kurdish
```

Comment:
We might have to do this for all the flag items, as depending on the OS + Browser combination it might render as the letters instead:

![image](https://github.com/zeeguu/web/assets/17390076/457753ad-8ed7-4657-b5b1-e18434b98c35)

In my case, Chrome + Windows, I see the letters.

Reply:
I didn't know Windows doesn't render flag emojis – good to know! I've changed all the emojis to images now and tested in both Chrome and Safari.